crt_portal/cts_forms/templates/forms/complaint_view/show/attachments.html

```diff
@@ -0,0 +1,34 @@
+{% extends "forms/complaint_view/show/card.html" %}
+{% load static %}
+{% block title %}{{title}}{% endblock %}
+{% block extra_classes %}crt-action-card{% endblock %}
+{% block icon %}
+<img src="{% static icon %}" alt="" class="icon" />
+{% endblock %}
+
+{% block card_content %}
+<form id="complaint-view-attachments" class="usa-form" method="post" action="{% url 'crt_forms:save-report-attachment' report_id=data.pk %}" enctype="multipart/form-data" novalidate>
+  {% csrf_token %}
+  <fieldset name="view-attachments" class="usa-fieldset usa-prose">
+    <legend class="usa-sr-only">Complaint detail view attachments</legend>
+    <input type="hidden" value="{{ return_url_args }}" name="next" id="next" />
+    <input type="hidden" value="{{ index }}" name="index" id="index" />
+    <input type="hidden" value="{{ data.pk }}" name="report" id="report_id" />
+    {% for attachment in attachments %}
+      <p>{{ attachment.filename }}</p>
+    {% endfor %}
+    <div class="intake-section">
+      
+      <input type="file" name="file" id="file_input" hidden/>
```

Comment:
Opportunity here to specify the filetypes we're accepting on the browser side as well -- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#limiting_accepted_file_types